wpe/src/main/cpp/Browser/Page.cpp

```diff
@@ -272,8 +278,11 @@ struct wpe_android_view_backend_exportable_client Page::s_exportableClient = {
         Browser::getInstance().handleExportedBuffer(*page, std::make_shared<ExportedBuffer>(buffer, poolID, bufferID));
     }};
 
-Page::Page(int width, int height, std::shared_ptr<PageEventObserver> observer)
-    : m_width(width)
+Page::Page(const std::string dataDir, const std::string cacheDir, int width, int height,
```

Comment:
`std::string` and NOT `const std::string` just like I wrote in my previous comment